site/docs/latest/advanced/PartitionedTopics.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Partitioned topics
+lead: Expand message throughput by distributing load within topics
+tags:
+- topics
+- partitioning
+- admin 
+- clients
+---
+
+By default, Pulsar {% popover topics %} are served by a single {% popover broker %}. Using only a single broker, however, limits a topic's maximum throughput. *Partitioned topics* are a special type of topic that can span multiple brokers and thus allow for much higher throughput. For an explanation of how partitioned topics work, see the [Concepts](#concepts) section below.
+
+You can [publish](#publishing-to-partitioned-topics) to partitioned topics using Pulsar's client libraries and you can [create and manage](#managing-partitioned-topics) partitioned topics using Pulsar's [admin API](../../admin/AdminInterface).
+
+## Publishing to partitioned topics
+
+When publising to partitioned topics, the only difference from non-partitioned topics is that you need to specify a [routing mode](../../getting-started/ConceptsAndArchitecture#routing-modes) when you create a new {% popover producer %}. Examples for [Java](#java) are below.
```

Comment:
As above:
> you need to specify

You're not forced to do it. Also, it should be more prominent that specifying a key will use the key hash to select the partition, guaranteeing the per-key ordering. (this is the most common pattern when using partitioned topics).